examples/ecs/service-static-cluster.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: ecs.aws.upbound.io/v1beta1
+kind: Service
+metadata:
+  name: example-service
+  annotations:
+    meta.upbound.io/example-id: ecs/v1beta1/service-static-cluster
+    upjet.upbound.io/manual-intervention: "Resource stuck in update loop"
```

Comment:
The example is uptest green, should we remove `upjet.upbound.io/manual-intervention` annotation?

Reply:
its added because of update loop check this: https://github.com/upbound/provider-aws/issues/585

Reply:
Ok, then
```suggestion
    upjet.upbound.io/manual-intervention: "Resource stuck in update loop, see https://github.com/upbound/provider-aws/issues/585"
```

But I am still confused because [uptest run](https://github.com/upbound/provider-aws/actions/runs/4588536248) is nicely green

Reply:
the resource overall is True / True all the time - but the update loop is running all the time / every loop